src/converse-core.js

```diff
@@ -605,6 +600,7 @@
                 _converse.clearMsgCounter();
             }
             _converse.windowState = state;
+            _converse.emit('windowStateChanged', {state: state})
```

Comment:
Please document any new events you create here: https://github.com/jcbrand/converse.js/blob/master/docs/source/events.rst